test/checks/language/valid-lang.js

```diff
@@ -43,13 +43,12 @@ describe('valid-lang', function () {
 			assert.deepEqual(checkContext._data, ['lang="en-FOO"']);
 		});
 
-		it('should return true (and not throw) when given no options', function () {
+		it('should return false (and not throw) when given no options', function () {
```

Comment:
Add a test that tests to make sure it returns true if passed an empty options array